CONTRIBUTING.md

```diff
@@ -138,10 +138,8 @@ After a few moments of initial configuration Visual Studio Code will launch the
 ### Live testing
 
 Before running or recording live tests you need to create
-[live test resources](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/TestResources/README.md).
-If recording tests, secrets will be sanitized from saved recordings.
-If you will be working on contributions over time, you should consider
-persisting these variables.
+[live test resources](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/TestResources/README.md).  If you will be working on contributions over time, you should consider
```

Comment:
Now that the `-OutFile` option is used by default, this sentence may not make sense any longer. The reason being that the .env file created by the new resource script now automatically persists the variables for as long as you are using a given cloned repo.

Reply:
hrmm.... fair point.   I'll pull this out and let the link to the test resources docs stand on their own.